tests/core/full_node/stores/test_block_store.py

```diff
@@ -367,3 +367,61 @@ async def test_get_block_bytes_in_range(tmp_dir: Path, bt: BlockTools, db_versio
 
             with pytest.raises(ValueError):
                 await store_2.get_block_bytes_in_range(0, 10)
+
+
+@pytest.mark.asyncio
+async def test_get_plot_filer_info(
+    default_1000_blocks: List[FullBlock], tmp_dir: Path, db_version: int, bt: BlockTools
+) -> None:
+    assert sqlite3.threadsafety >= 1
+
```

Comment:
I'm keeping consistency with the above tests.

Reply:
Not sure if other tests doing something unnecessary justifies continuing doing it? :) If this is something we want to test it should be some separate test which tests this rather imo since its a global thing and there is no point to copy/paste it into all tests.